client/scss/base/_row.scss

```diff
@@ -25,6 +25,10 @@
   padding-bottom: 0;
 }
 
+.row--small-pad-top {
```

Comment:
`.row--small-top-padding` would be in keeping with the `--no-bottom-padding` modifier.